src/solid/scope.ts

```diff
@@ -58,8 +58,9 @@ export function useGate<Props>(
 
   createEffect(() => commonUseGate(ForkedGate(), props))
 }
-export function useUnit(shape) {
-  return useUnitBase(shape, getScope())
+export function useUnit(shape, opts?: {forceScope?: boolean}) {
+  const scope = getScope(opts?.forceScope)
```

Comment:
I think that `/scope` version of bindings should have `forceScope` enabled by default 🤔 

I suggest to change it like that:
```suggestion
  const scope = getScope(opts?.forceScope ?? true)
```

Reply:
Hmm, i thougth a little more and now i suggest to do it like this:


1. Inside `getScope` there should be default fallback like `forceScope = true`

2. For `/scope` exports we don't need to do anything besides `getScope()` call without params

3. For main `effector-solid` exports we should do something like `getScope(ops?.forceScope ?? false)` inside